Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Race Trait Adaption

## Changelog:
- Added all the race traits back (so far no effects, just placeholders).
- Every race trait has localisation and icon.

## How to test:
Enter the game, open the Character Finder, check character traits. Tell your opinion on icons and texts.

If you're a @Warcraft-GoA-Development-Team/english-watchers, please proofread the `.yml` files.
